src/main.cpp

```diff
@@ -418,40 +638,50 @@ void emulator_loop()
 		//
 		// Trace functionality preserved for comparison with official emulator releases
 		//
-#if defined(TRACE)
+//#if defined(TRACE)
```

Comment:
I'm definitely going to want to preserve `#if defined(TRACE)` for performance when folks don't need trace output. Folks who want a trace build can currently do a `make MYFLAGS="-DTRACE"`, though I'd agree that it might be worthwhile to add a make target to include `-DTRACE` in the compiler flags.

Reply:
OK! Agree! These if statements on the addresses take CPU time and commenting this block will solve this. It is a solution to make 2 versions, one with and one without trace.